src/reset.scss

```diff
@@ -0,0 +1,3 @@
+body {
```

Comment:
Consider using a more comprehensive CSS reset or normalize for better cross-browser consistency.